app-scroll-effects/app-scroll-effects.html

```diff
@@ -62,20 +84,25 @@
         this.$.backgroundRearLayer.style.opacity = 0;
       }
     }
-  };
+  },
 
   /**
    * Vertically translate the background based on a factor of the scroll position.
    */
-  scrollEffects['parallax-background'] = {
+  'parallax-background': {
+    /**
+     * @param {{scalar: string}} config
+     * @this ElementWithBackground
+     */
     setUp: function setUp(config) {
       var scalar = parseFloat(config.scalar);
+
       this._deltaBg = this.$.backgroundFrontLayer.offsetHeight - this.$.background.offsetHeight;
       if (this._deltaBg === 0) {
         if (isNaN(scalar)) {
           scalar = 0.8;
         }
-        this._deltaBg = this._dHeight * scalar;
+        this._deltaBg = this._deltaHeight * scalar;
```

Comment:
It's not going through this path right now, but app-box should have a `dHeight`. Maybe dHeight could be: `dHeight = this.$.backgroundFrontLayer.offsetHeight - this.$.background.offsetHeight` and now we might be able to refactor this code a bit.